instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java

```diff
@@ -72,6 +84,22 @@ public Builder remoteServiceName(String remoteServiceName) {
       return this;
     }
 
+    /**
+     * Opt-out of sharing poll span when no trace-context or extra propagation field is available on
+     * incoming messages.
+     *
+     * <b/>If true, a poll root span will be created for all messages without trace-context, and all
```

Comment:
`If true, all the spans received in a poll batch that do not have trace-context will be added to a single new poll root span. Otherwise, a poll span will be created for each such message.`